storage/range_tree_test.go

```diff
@@ -31,8 +31,9 @@ import (
 // SetupRangeTree is called via store.BootstrapRange.
 func TestSetupRangeTree(t *testing.T) {
```

Comment:
I looked at both that one and also the original go implementation https://github.com/petar/GoLLRB/blob/master/llrb/llrb.go